Relationships/RelationshipResolver.cs

```diff
@@ -123,22 +145,22 @@ private void Resolve(CkanModule module, RelationshipResolverOptions options)
             // Even though we may resolve top-level suggests for our module,
             // we don't install suggestions all the down unless with_all_suggests
             // is true.
-            var sub_options = (RelationshipResolverOptions)options.Clone();
+            var sub_options = (RelationshipResolverOptions) options.Clone();
             sub_options.with_suggests = false;
 
             log.DebugFormat("Resolving dependencies for {0}", module.identifier);
-            ResolveStanza(module.depends, sub_options);
+            ResolveStanza(module.depends, new Relationship.Depends(module), sub_options);
```

Comment:
Noting to self: if we're processing dependencies, we have a 'hard' resolve (ie: it must complete).